ci/taos/plugins-good/pr-prebuild-clang.sh

```diff
@@ -56,10 +56,14 @@ function pr-prebuild-clang(){
     echo "[DEBUG] Files to be tested: $FILES_TO_BE_TESTED"
 
     # import clang configuration file
-    ln -sf ci/doc/.clang-format .clang-format
+    if [ ! -f ".clang-forma" ]; then
```

Comment:
.clang-format

Reply:
Oooops. Thanks.

Reply:
@gichan-jang , Excellent. 🥇 . It seems that you get the Hawk Eye. :) 

@jijoongmoon,  Please use [[ ]] to avoid unexpected issue (e.g., unary issue).
* before: if [ ! -f ".clang-format" ]; then
* after: if [[ ! -f ".clang-format" ]]; then